src/main/java/frc/robot/commands/AimWithLimelight.java

```diff
@@ -155,9 +155,9 @@ public void execute() {
       /* Remember if we've seen a target so that we can assume we're aimed if we lose sight of the target
        * Only reason we assume this is because of where the Limelight is, which results in us not actually seeing the target if we're properly aimed
        */
-      hasSeenTarget = true;
+      // hasSeenTarget = true;
```

Comment:
remove commented code

Reply:
unless it needs to be there

Reply:
Has to stay there because it was a quickfix that got reverted but not completely. It'll have to get fixed next season.

Reply:
Or more accurately ig: Limelight is a disaster and doesn't work. We tried to fix it. The fix didn't work. We don't know if we'll need the fix again later.